### DISCUSSION TITLE
3D 리소스의 최대 제약은 어느정도인가요?

### DISCUSSION THREAD

**Discussion by Astoration** [naverz/zepeto-studio-kor#732] [5. Q&A]
수용 가능한 폴리곤, 오브젝트 개수는 유니티와 동일하게 따라가는걸까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 네 맞습니다. 별도의 제약은 없으나 최적화 및 용량 제한만 지켜주시면 문제 없습니다~
https://github.com/naverz/zepeto-studio-kor/discussions/695
최저 사양 참고 부탁 드립니다.
감사합니다.
